apps/public-docsite/src/pages/Styles/FabricIconsPage/docs/web/FabricIconsUsage.md

```diff
@@ -1,4 +1,27 @@
-You can use Fluent UI's icons in a few ways, depending on if you're using Fluent UI React or Fabric Core.
+You can use Fluent UI's icons in a few ways, depending on if you're using React Icons, Fluent UI React or Fabric Core.
+
+### React Icons: SVG Based React Icons
```

Comment:
nit, Microsoft style is to use sentence case (though we're very inconsistent about it today)
```suggestion
### React Icons: SVG-based React icons
```